metaflow/cli.py

```diff
@@ -969,6 +975,10 @@ def start(
     ctx.obj.monitor.start()
     _system_monitor.init_system_monitor(ctx.obj.flow.name, ctx.obj.monitor)
 
+    # Add package suffixes and echo to the flow object as they are used by decorators
```

Comment:
This seems not needed anymore since you pass it to the MetaflowPackage constructor.